src/services/profileService.ts

```diff
@@ -136,7 +136,9 @@ export default class ProfileService {
     | IResponseErrorInternal
     | IResponseErrorTooManyRequests
     | IResponseErrorConflict
-    | IResponseSuccessJson<InitializedProfile>
+    // This Service response is not binded with any API, so we remove any payload
+    // from this Response Success JSON.
+    | IResponseSuccessJson<unknown>
```

Comment:
To describe an empty object we could use `Record<string, never>` instead of `unknown`

Reply:
Fixed in [2fa58c9](https://github.com/pagopa/io-backend/pull/993/commits/2fa58c99d672979d4984e5a13747313157dd1604)